tests/detect-ldap-dn/test.rules

```diff
@@ -0,0 +1,2 @@
+alert ldap any any -> any any (msg:"Test ldap request dn"; ldap.request.distinguished_name; content:"dc=example,dc=com"; startswith; endswith; sid:1;)
+alert ldap any any -> any any (msg:"Test ldap responses dn"; ldap.responses.dn; content:"dc=example,dc=com"; startswith; endswith; sid:2;)
```

Comment:
let's use `distinguished_name` or `dn` consistently for both request and responses